packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -44,10 +44,14 @@ type FeedQueryProps = {
 };
 
 const propsByFeed: Record<string, FeedQueryProps> = {
-  popular: {
+  'my-feed': {
     query: ANONYMOUS_FEED_QUERY,
     queryIfLogged: FEED_QUERY,
   },
+  popular: {
+    query: ANONYMOUS_FEED_QUERY,
+    queryIfLogged: ANONYMOUS_FEED_QUERY,
```

Comment:
`queryIfLogged` should be dependent on the A/B. If the A/B is off then it should be `FEED_QUERY`